cmd/cli/osm.go

```diff
@@ -77,4 +79,11 @@ func main() {
 }
 
 func debug(format string, v ...interface{}) {
+	if settings.Debug {
+		format = fmt.Sprintf("[debug] %s\n", format)
+		err := log.Output(2, fmt.Sprintf(format, v...))
```

Comment:
Instead of `log`, `fmt.Printf()` should suffice in the CLI.

Reply:
`fmt.Printf()` works when I tested but it doesn't give the date/time like `log` does.

Reply:
That's expected. Given this is a CLI command, we should preferably printing to stdout/stderr instead of using a logger.